lib/qiita/markdown/filters/inline_code_color.rb

```diff
@@ -29,7 +31,11 @@ def call
         private
 
         def color_element(color)
-          %(<span class="" style="background-color: #{color};"></span>)
+          %(<span class=#{inline_code_color_class_name} style="background-color: #{color};"></span>)
+        end
+
+        def inline_code_color_class_name
+          @options ||= context[:inline_code_color_class_name] || DEFAULT_CLASS_NAME
```

Comment:
In general, instance variables for caching should be given the same name as the method name. BTW, I think there is no need to cache `#inline_code_color_class_name` since it is so trivial. 🤔

Reply:
Certainly it does not seem necessary to cash. Thank you!